### PR TITLE
fix is_active template filter [#3077][#3429]

### PR DESCRIPTION
**Pull request by devincowan**
This PR fixes a regression introduced in PR #4576 

### Pull Request Checklist: 
- [x] Positive Test Case Written by Dev

- [ ] Automated Testing

- [x] Sufficient User and Developer Documentation

- [x] Passing Jenkins Build

- [x] Peer Code review and approval

### Positive Test Case
1. Create a resource
2. Add an author to a resource who is not a hydroshare user
3. Reload the page -- no longer get a template error on page reload

All of the tests in PR #4576 still hold true

### CODE REVIEW

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4627](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4627)

File: hs_core/page_processors.py [hunk 1]
```diff
@@ -118,6 +118,11 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
     topics = list(topics)  # force QuerySet evaluation
     content_model.update_relation_meta()
     creators = content_model.metadata.creators.all()
+    has_active_creators = False
+    for creator in creators:
+        if creator.is_active:
```
Comment (pkdash) [inline hs_core/page_processors.py:123]: This raises an exception if the first creator is not a hydroshare user. You can test by rearranging/re-ordering the authors in UI making the non-hydroshare user as the first author.

**Review by pkdash — CHANGES_REQUESTED** [review]

File: hs_core/page_processors.py (hunk 1 above)
Comment (devincowan) [reply hs_core/page_processors.py:123]: Correct, thank you. Modified to catch the exception now that I understand better...

File: hs_core/templates/pages/genericresource.html [hunk 2]
```diff
@@ -1019,7 +1019,6 @@ <h3>Resource Specific</h3>
                     "profileUrl": "{{ author.relative_uri|default:'' }}",
```
[the remark this answers is not in the document]
Comment (devincowan) [reply hs_core/templates/pages/genericresource.html:1019]: You're right, 4fe65f7bcbaad960939051c38520bc51a0292db4 should fix this too. Thanks

File: hs_core/models.py [hunk 3]
```diff
@@ -613,15 +613,15 @@ def update(cls, element_id, **kwargs):
...
     @property
     def is_active(self):
         from hs_core.hydroshare.utils import user_from_id
-        user = user_from_id(self.hydroshare_user_id, raise404=False)
-        if user:
+        try:
+            user = user_from_id(self.hydroshare_user_id, raise404=False)
             return user.is_active
-        else:
+        except:
```
Comment (pkdash) [inline hs_core/models.py:624]: Should we here catch only `ObjectDoesNotExist`?

**Review by pkdash — APPROVED** [review]
Left a comment for your consideration.

**Review by pkdash — APPROVED** [review]

**Pull request merged**